api/graphql/schema.graphql

```diff
@@ -439,7 +439,7 @@ input UpdatePoolCandidateSearchRequestInput {
 }
 
 type Mutation {
-    createUser(user: CreateUserInput! @spread): User @create @guard @can(ability: "create")
+    createUser(user: CreateUserInput! @spread): User @create @guard @can(ability: "create") @populateSub(source: "email")
```

Comment:
Would it make more sense to have a generic "copy field a to b" directive?  Or is "sub" going to be a standardized column that so it is useful to have its own directive?

Reply:
Or even less generic?  If it will always be email to sub then make the directive do that with no arguments?

Reply:
I really like the idea of the more generic, copy a to b, version

Reply:
Committed a new version that can copy from a to b